<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QWidget" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLabel" name="labelWelcome">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Witaj, &quot;Imie&quot;!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButtonDodajSaldo">
     <property name="text">
      <string>Dodaj saldo</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="pushButtonWrzucButelki">
     <property name="text">
      <string>Wrzuć butelki</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QPushButton" name="pushButtonWyplacSaldo">
     <property name="text">
      <string>Wypłać saldo</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="labelSaldo">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0.00 zł</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="labelTwojeSaldo">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Twoje saldo:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Historia transakcji:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QListWidget" name="listWidgetHistoria"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
